config/routes.rb

```diff
@@ -2,9 +2,10 @@
   root 'homes#index'
   devise_for :users
 
-  resources :vendors, only: [:index, :show, :edit, :update, :create] do
+  resources :vendors do
     resources :reviews
   end
 
   resources :locations
```

Comment:
You should scope your routes to only the ones being used. Otherwise `rake routes` outputs garbage.